### ISSUE TITLE
Issue with ElseIf

### ISSUE THREAD

**Issue by Mummified316** [Pulover/PuloversMacroCreator#137]
In the event that an `If` statement has a second (Or more) `If` statement nested within it, and that second `If` statement has an `ElseIf`, and the first `If` statement resolves as false than the macro will terminate.

Example (The MessageBox never runs):
```
If (1 = 2)
{
    If (1 = 1)
    {
    }
    Else If (1 = 1)
    {
    }
}
MsgBox, 0, , Test text.
```

**Comment by Pulover** [comment]
Will fix that. Thanks!

**Comment by Pulover** [comment]
Fixed in v5.3.3.
